src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (currentTab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab: Tab = (
+    tabs.find((tab) => (
+      tab.id === selectedTabId
+    )) || tabs[0]
+  );
+  const handleTabSelect = (
+    currentTabId: string,
```

Comment:
You don't need to pass this id as an argument. You have it already `selectedTabId`